great_expectations/core/expectation_configuration.py

```diff
@@ -1247,6 +1251,7 @@ def isEquivalentTo(self, other, match_type="success"):
                     self.kwargs == other.kwargs,
                 )
             )
+        return False
```

Comment:
This method has a bunch of `elif` but no `else`. Instead of returning `None`, I've opted to return `False` here.

I've reviewed all usages of this method and believe they just check the truthy/falsy-ness of the result; thus, the end result should be the same.